english/mystory.md

```diff
@@ -0,0 +1,7 @@
+Once a cat was caught in a hunter's net. A mouse used to live in a nearby hole. The mouse seeing the cat in the net started playing around the cat. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.